analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -468,8 +468,11 @@ def check(check_data):
 
         result_file = ''
 
+        if analyzer_config is None:
```

Comment:
```suggestion
        if not analyzer_config:
```

Reply:
I know we can check that this way too but they do not mean exactly the same check.
is None means the value is not set an in the other case some value can be set but it will be evaluated to true or false.
The two expressions are almost the same but do not mean the same thing as mentioned in the [programming recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) too.

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.
> 
> Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

I wanted to express in the code that the analyzer_config is not set.